README.md

```diff
@@ -1,17 +1,17 @@
 # Akri and Krustlet integration
-This project contains [Akri](https://github.com/deislabs/akri) components built and designed to be executed as a [WebAssembly](https://webassembly.org/) application inside kubernetes with [Krustlet](https://github.com/deislabs/krustlet).
+This project contains [Akri](https://github.com/deislabs/akri) components built and designed to be executed as a [WebAssembly](https://webassembly.org/) application inside Kubernetes with [Krustlet](https://github.com/krustlet/krustlet).
 
 ## About
 
-Akri is a Kubernetes component that finds and lets you use IoT devices under the k8s environment. Krustlet is also a Kubernetes component and lets you run WebAssembly application (In addition to docker containers) on your k8s cluster. This project is about merging these two ideas together and making Akri compiled as a wasm-wasi application be deployed into a Kubernetes cluster using Krustlet, creating new ways to use these devices in your WASM applications and give Akri users all the advantages of this technology.
+Akri is a Kubernetes component that finds and lets you use IoT devices under the k8s environment. Krustlet is also a Kubernetes component and lets you run WebAssembly application (In addition to docker containers) on your k8s cluster. This project is about merging these two ideas together and making Akri compiled as a wasm-wasi application be deployed into a Kubernetes cluster using Krustlet, creating new ways to use these devices in your Wasm applications and give Akri users all the advantages of this technology.
 
-As it was designed to be runed on the edge, Akri is not expected to consume lots of memory and could benefit from any improvements in this aspect. By making Akri compatible with Krustlet a significant improvement in startup time, runtime performance and memory consumption is expect, reducing the consumption of the user's resources.
+As it was designed to be executed on the edge, Akri is not expected to consume lots of memory and could benefit from any improvements in this aspect. By making Akri compatible with Krustlet a significant improvement in startup time, runtime performance and memory consumption is expected, reducing the consumption of the user's resources.
 
 ### WebAssembly (Wasm)
 
-Wasm has been a promising tool and has already revolutionized web development, but all the predictability, scalability, efficiency, and security improvements that it has when compared with other browsers tools like JavaScript could also be applied to server-side technologies like servers, services and . 
+Wasm has been a promising tool and has already revolutionized web development, but all the predictability, scalability, efficiency, and security improvements that it has when compared with other browsers tools like JavaScript could also be applied to server-side technologies like servers and services. 
```

Comment:
nit: "to server-side technologies like servers and services" seems a bit redundant. Maybe end at technologies